particles/Tutorial/Kotlin/Demo/README.md

```diff
@@ -0,0 +1,89 @@
+# Behold Tic Tac Toe
+
+## Articles about Designing with Particles
+
+So from our introductory tutorials, we have the following definition of a particle:
+
+> *Particle* - Modular component of functionality. Ideally small units so particles can be reusable. 
+
+While that tells us _what_ a particle is, it does not say _how_ a particle should be used and _how_
+a recipe should be divided into particles. To better understand how to design a system in Arcs, it
+is important to have a full mental picture of particles. Just as there are multiple sides to any 
+person, particles have multiple faces. To fully understand them, we must look below the surface to
+see the soul of particles.
+
+#### Smallest Components
+Even from the definition of particles, we know they should be small components. Just as with object
+oriented programming, this lets us have easily interchangeable components. One can envision a world
+where particles are made by external developers and combined to form recipes - much the way 
+developers use external libraries in object-oriented languages.
+
+However, making particles as small as possible leads us towards a "turtles all the way down"
+mentality. There is obvious overhead involved in each particle, therefore the desire for small
+particles must be balanced against having too many.
+
+#### Bounds of Data Knowledge
+Arcs' mission statement says that users have sovereignty over their data and its use. To ensure
+data only travels as intended, Arcs uses data flow analysis to make grantees about data flow. But,
+for this to work properly, particles must delineate knowledge boundaries. Just as each nation has a 
+border so customs and immigration can know where to operate, we need to create systems with well 
```

Comment:
I like the analogy used here, but I wonder if an immune system or just a company, made up of teams, might be less political? Just an idea rather than a requested change